src/main/java/core/basesyntax/FootballMatch.java

```diff
@@ -15,6 +15,25 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        int guess;
+        int maxRes;
+        int maxBet;
```

Comment:
good start, but you can make this solution shorter and easier to understand.
let's try to solve this task without creating new variables

and don't forget about checking yourself before submitting task
https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_01.html